cmd/sidecar/main.go

```diff
@@ -40,17 +46,33 @@ func main() {
 		os.Exit(1)
 	}
 
-	configSecrets, err := secretsConfig.NewFromEnv(tokenFilePath)
+	configStorage, err := storageConfig.NewFromEnv()
 	if err != nil {
 		errLogger.Printf(err.Error())
 		os.Exit(1)
 	}
+	// Create new Storage
+	stor, err := storage.NewStorageHandler(*configStorage)
+
+	//if conjurSecretsDest == 1  &&  authn.Config.ContainerMode != "init" {
+	//      // notify on invalid configuration and exit
+	//      errLogger.Printf(" appropriate message for not supporting sidecar ....")
+	//      os.Exit(1)
+	//}
+
+	if conjurSecretsDest == 1 {
+		configSecrets, err := secretsConfig.NewFromEnv(tokenFilePath)
```

Comment:
@orenbm @nessiLahav @moticless I know we are slowing moving this logic over to different files but is this logic correct here? if i am understanding correctly, if `secrets_destination===1` (k8s) then read from token in set path. This is not what we want here b/c we want to read the access token from memory. correct?